src/MSDF.DataChecker.WebApi/Properties/launchSettings.json

```diff
@@ -6,21 +6,15 @@
             "launchBrowser": true,
             "applicationUrl": "http://localhost:5500;",
             "environmentVariables": {
-                "DOTNET_ENVIRONMENT": "Development",
-                "ConnectionStrings:DataCheckerStore": "Data Source=localhost; Database=DataChecker; Integrated Security=true; Persist Security Info=True;",
-                "ConnectionStrings:RulesExecutorStore": "Data Source=localhost; Database=DataChecker; Integrated Security=true; Persist Security Info=True;",
-                "DatabaseEngine": "SqlServer"
+                "ASPNETCORE_ENVIRONMENT": "SQL-Development"
```

Comment:
i prefer not to have the app settings broken apart, and rather use the overrides as documentation. secondly when we go to docker, this will be a moot point. Third the app settings are defined using inheritance, so we should continue with that pattern.